### PR TITLE
factor out sc_isolation_enrichment

### PR DESCRIPTION
**Pull request by mccalluc**
Use the suggest-edit mechanism to indicate which is the correct definition for this field; Chuck will followup and regenerate docs.

### CODE REVIEW

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_enrichment.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# scrnaseq-hca-v0.yaml
+- name: sc_isolation_enrichment
+  constraints:
+    required: true
```
Comment (cebriggs7135) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_enrichment.yaml:4]:
```suggestion
       enum:
           - none
           - FACS
```

Comment (cebriggs7135) [reply src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_enrichment.yaml:4]: This should have the same enums as the other sc assays.

**Review by cebriggs7135 — CHANGES_REQUESTED** [review]
Give HCA the same enums for this field.

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_enrichment.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  constraints:
+    enum:
+      - none
+      - FACS
+
+# scrnaseq-v0.yaml
+- name: sc_isolation_enrichment
+  description: The method by which specific cell populations are sorted or enriched.
+  constraints:
+    required: False
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_enrichment.yaml:19]:
```suggestion
```

File: src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_enrichment.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# scrnaseq-hca-v0.yaml
+- name: sc_isolation_enrichment
+  constraints:
+    required: true
+  description: The method by which specific cell populations are sorted or enriched.
+
+# scatacseq-v0.yaml; scatacseq-v1.yaml; scrnaseq-v1.yaml; scrnaseq-v2.yaml
```
Comment (mccalluc) [inline src/ingest_validation_tools/table-schemas/includes/fields/sc_isolation_enrichment.yaml:7]:
```suggestion
```

**Pull request closed without merging**